### ISSUE TITLE
Feedback on spans

### ISSUE THREAD

**Issue by danmargalit** [QBobWatson/gt-linalg#26]
In reference to version a327fdb0a331919e37453b572dd7d516b8205a2f

We say there are two ways of thinking about linear systems: (1) augmented matrices and (2) vector equations.  Why not say we have three ways of thinking of the same idea: (1) linear systems, (2) augmented matrices, and (3) vector equations.  We should also emphasize that the reader should get comfortable going from any of of these forms to any other.

Again, happy to make edits.  I'm a little uncomfortable making such changes without an ok.

**Comment by QBobWatson** [comment]
Good idea -- maybe also put an example system (like the one from the beginning of the section), as in section 3.3?
